modules/web-console/frontend/app/modules/agent/AgentManager.service.ts

```diff
@@ -176,8 +177,16 @@ class ConnectionState {
 export default class AgentManager {
     static $inject = ['Demo', '$q', '$transitions', '$location', 'AgentModal', 'UserNotifications', 'IgniteVersion', 'ClusterLoginService'];
 
+    currentCluster$: Observable<ConnectionState>;
+    clusterIsActive$: Observable<boolean>;
+    clusterIsAvailable$: Observable<boolean>;
+
     clusterVersion: string;
 
+    indexing: boolean;
```

Comment:
Do we need  `indexing`, `ruStaus` and `snapshoChainMode`? I think we should have only `supportedFeatures` and enum or bit numbers to later check with method like `isFeatureSupported(features, FEATURE)`

Reply:
I agree with @akuznetsov-gridgain, there's too much going on AgentManager already, we can at least reduce the amount of properties it exposes.